src/Jackett.Common/Definitions/uploadcx.yml

```
@@ -1,7 +1,7 @@
 ---
 id: uploadcx
 name: upload.cx
-description: "upload.cx is a Private Torrent Tracker for MOVIES"
+description: "upload.cx is a Private Torrent Tracker for MOVIES & TV"
```

```suggestion
description: "upload.cx is a Private Torrent Tracker for MOVIES / TV"
```
conform to common Jackett description standards